swaybar/event_loop.c

```diff
@@ -20,6 +20,19 @@ struct timer_item {
 	void *data;
 };
 
+enum state_item_flags {
+	ITEM_IS_FD = 0 << 0,
+	ITEM_IS_TIMER = 1 << 0,
+};
```

Comment:
These should be `1 << 0` and `1 << 1` if this is trying to be a bitmask.
But seeing as it's a tag for a tagged union, I don't think that's actually what you're going for, so there would be no reason for the weird `<< 0`.

Reply:
For awhile I was thinking about making this a bitmask so different operations could be added to the flags (change mask, add event, etc), but now there isn't any of that so I'll remove it